phpBB/includes/functions_upload.php

```diff
@@ -294,7 +294,7 @@ function check_content($disallowed_content)
 	*/
 	function move_file($destination, $overwrite = false, $skip_image_check = false, $chmod = false)
 	{
-		global $user, $phpbb_root_path;
+		global $user, $phpbb_root_path, $mode, $phpbb_dispatcher;
```

Comment:
please add `$mode` as an argument with whatever default value at the end of the argument list.